skins/laika/src/components/pages/donation_form/subpages/AddressPage.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+	<div class="address-page">
+		<address-fields v-bind="$props"></address-fields>
+		<div class="column">
+			<div class="summary-wrapper">
+				<donation-summary :payment="paymentSummary" :address-type="addressType" :address="addressSummary"></donation-summary>
+				<submit-values :tracking-data="{}"></submit-values>
+				<div class="level has-margin-top-36">
+					<div class="level-left">
+						<b-button id="previous-btn" class="level-item"
+								  @click="$emit( 'previous-page' )"
+								  type="is-primary is-main"
+								  outlined>
+							{{ $t('donation_form_section_back') }}
+						</b-button>
+					</div>
+					<div class="level-right">
+						<b-button id="submit-btn" class="level-item"
+								  @click="submit"
+								  type="is-primary is-main">
+							{{ $t('donation_form_finalize') }}
+						</b-button>
+					</div>
+				</div>
+			</div>
+		</div>
+
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { addressTypeName } from '../../../../view_models/AddressTypeModel';
+import { NS_ADDRESS, NS_PAYMENT } from '@/store/namespaces';
+import AddressFields from '@/components/pages/donation_form/Address.vue';
+import DonationSummary from '@/components/DonationSummary.vue';
+import SubmitValues from '@/components/pages/donation_form/SubmitValues.vue';
+import { TrackingData } from '@/view_models/SubmitValues';
+
+export default Vue.extend( {
+	name: 'AddressPage',
+	components: {
+		AddressFields,
+		DonationSummary,
+		SubmitValues,
+	},
+	props: {
+		validateAddressUrl: String,
+		addressCountries: Array as () => Array<String>,
+		trackingData: Object as () => TrackingData,
+	},
+	computed: {
+		paymentSummary: {
+			get(): object {
+				const payment = this.$store.state[ NS_PAYMENT ].values;
+				return {
+					interval: payment.interval,
+					amount: payment.amount / 100,
+					paymentType: payment.type,
+				};
+			},
+		},
+		addressType: {
+			get(): string {
+				return addressTypeName( this.$store.state[ NS_ADDRESS ].addressType );
+			},
+		},
+		addressSummary: {
+			get(): object {
+				return {
+					...this.$store.state[ NS_ADDRESS ].values,
+					fullName: this.$store.getters[ NS_ADDRESS + '/fullName' ],
+					streetAddress: this.$store.state[ NS_ADDRESS ].values.street,
+					postalCode: this.$store.state[ NS_ADDRESS ].values.postcode,
+					countryCode: this.$store.state[ NS_ADDRESS ].values.country,
+				};
+			},
+		},
+	},
+	methods: {
+		submit() {
+			// TODO validate address, then submit form
+		},
+	},
+} );
+</script>
+
+<style lang="scss" scoped>
+	@import "../../../../scss/custom";
+
+	.summary-wrapper {
```

Comment:
@gbirke I pushed another commit where I have updated the `$column-gap` variable to 0.65rem which effectively makes the column padding 18px. I have also changed the headline string and updated the size in the review box to match the Figma design. It looks good to me now but maybe double-check if I missed anything.

I just noticed, though, that the country selection is empty on my home computer. Is this a configuration issue? It was working fine on my notebook at work earlier if I am not mistaken, but I am not sure if this may be a newly introduced issue.

Reply:
I fixed the country issue introduced in my last commit.

The headline string is different for donation review and donation confirmation, so I'll make another commit putting that in a slot.